book/go/getting-started/macos-setup.md

```diff
@@ -95,18 +95,20 @@ git checkout {{ book.wallaroo_version }}
 
 This will create a subdirectory called `wallaroo`.
 
-## Compiling Giles Sender, Data Receiver, and the Cluster Shutdown tool
+## Compiling Giles Sender, Data Receiver, Cluster Shutdown, and Cluster Shrinker tools
 
 Giles Sender is used to supply data to Wallaroo applications over TCP, and Data Receiver is used as a fast TCP Sink that can write the messages it receives to STDOUT. The two together are useful when developing and testing applications that use TCP Sources and a TCP Sink.
 
 The Cluster Shutdown tool is used to instruct the cluster to shutdown cleanly, clearing away any resilience and recovery files it may have created.
 
+The Cluster Shrinker tool is used to tell a running cluster to reduce the number of workers in the cluster and to query the cluster for information about how many workers are eligible for removal.
+
 To compile all three, run
 
 ```bash
 cd ~/wallaroo-tutorial/wallaroo/
 make build-giles-all build-utils-cluster_shutdown-all \
-  build-utils-data_receiver-all
+  build-utils-cluster_shrinker-all build-utils-data_receiver-all
```

Comment:
could these be `build-utils-all` to build them all?

definitely easier and less error prone to copy if it will build everything

Reply:
yes